packages/twenty-website/src/content/user-guide/functions/emails.mdx

```diff
@@ -42,7 +50,11 @@ From here, you can choose different levels of sharing for outbound and inbound e
 
 ### Contacts Auto-Creation
 
-Contact auto-creation is a handy built-in feature. This default feature automatically adds email contacts not already in your CRM, boosting your contact list without any extra effort on your part. To manage this feature, go to `Settings` > `Email`. You can toggle it on or off. Remember, turning it off means that all new email contacts must be manually entered into the CRM.
+Contact auto-creation is a handy built-in feature. This default feature automatically adds email contacts not already in your CRM, boosting your contact list without any extra effort on your part. To manage this feature, go to `Settings` > `Email`. You can toggle it on or off. Remember, turning it off means that all new email contacts must be manually entered into the CRM.  
+
+Deleted contacts are not re-imported. So if you delete a contact from Twenty, it will not be re-imported when receiving a new email from that contact.  
```

Comment:
Maybe explain they will not be reimported while they are soft-deleted? And avoid the repetition

Reply:
- I removed the duplicate
- "Soft delete" is something quite unclear for a majority of  users i believe. I try to avoid mentionning it but that's debatable